example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -467,6 +468,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.signify.huereactivebleExample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
+				SWIFT_VERSION = 5.0;
```

Comment:
Are change to this file important? This is the iOS "runner" project for the example app.

Also, this Swift version is different from the one declared in the pod-spec file for the library. Is 5.0 the highest Swift version the example app compiles with?

Reply:
I think this was purely done to reduce warnings in Xcode and declare the actually used version. 

5.5 is probably becoming 5.0 (when declared in .podspec) as this is more about the language level than a specific version.